Cargo.toml

```diff
@@ -8,17 +8,17 @@ version = "0.0.1"
 edition = "2021"
 
 [dependencies]
-any_spawner = { git = "https://github.com/leptos-rs/leptos",  features = ["futures-executor"] }
-throw_error = { git = "https://github.com/leptos-rs/leptos" }
-hydration_context = { git = "https://github.com/leptos-rs/leptos" }
+any_spawner = { git = "https://github.com/leptos-rs/leptos", rev = "84bdd6b56892cc859b512b0de1dc139f83375f5d" , features = ["futures-executor"] }
```

Comment:
Have you tried using the crate in a project after that? I remember when I tried doing that I had some trouble during compilation because `leptos` was using a different version for `any_spawner`

Reply:
Indeed, this does work in the leptos-spin template https://github.com/fermyon/leptos-spin/pull/48 -- I tested it by swapping the current pinned commit of leptos_wasi to my branch with `leptos_wasi = { git = "https://github.com/ogghead/leptos_wasi", branch = "rc1-locked-deps", optional = true }` and the Spin app spins up while referring to rc1 versions of Leptos crates